code/obj/machinery/floorflusher.dm

```diff
@@ -228,6 +228,11 @@
 					boutput(M, "You feel your handcuffs being removed.")
 					M.handcuffs.drop_handcuffs(M)
 
+				//Might as well set their security record to "released"
+				SPAWN(1)
```

Comment:
Why is this in a SPAWN?

Reply:
Long story short: I wanted to make sure the process wasn't causing enough lag to mess up the disposal work. In testing I've had crashes due to flusher sending me in the void when destroyed by flipping and wiggling. I realized this issue is in the main branch, however, so the SPAWN is no longer necessary indeed

Reply:
i'd remove it then

Reply:
Done, I also added a condition so only people on arrest or or incarcerated get the released status